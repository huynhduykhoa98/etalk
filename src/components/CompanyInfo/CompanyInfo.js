/* eslint-disable linebreak-style */
/* eslint-disable class-methods-use-this */
import React, { Component } from 'react';
import CompanyInfoShortCut from './CompanyInfoShortCut/CompanyInfoShortCut';

class CompanyInfo extends Component {
  render() {
    return (
      <CompanyInfoShortCut
        companyName="SPIDER COMPANY LIMITED"
        linkAddress="https://goo.gl/maps/h1ygYS3HS8xfzrPS8"
        titleAddress="Google maps"
        linkPhone="tel:+84905549909"
        titlePhone="Call us"
        linkContact="mailto:contact@spider.vn"
        titleContact="Contact us"
        linkWeb="//spider.vn"
        titleWeb="Visit our website"
        linkSkyofBoss="skype:len.nguyenvan?chat"
        titleSkyofBoss="Chat with us"
        // info
        Address=" 8 Hoang Minh Giam Street, Ward 9, Phu Nhuan District, Ho Chi Minh City, Vietnam"
        Mobile="+84 90 5549909"
        Email="contact@spider.vn"
        Website="spider.vn"
        Skype="len.nguyenvan"
      />
    );
  }
}

export default CompanyInfo;
