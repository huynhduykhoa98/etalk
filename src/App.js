/* eslint-disable quotes */
/* eslint-disable class-methods-use-this */
import React from 'react';
import { IntlProvider } from 'react-intl';
import Home from './components/Home/Home';
import Navbar from './components/Navbar/Navbar';
import About from './components/About/About';
import Service from './components/Service/Service';
import Technology from './components/Technology/Technology';
import Client from './components/Client/Client';
import Career from './components/Career/Career';
import Contact from './components/Contact/Contact';
import CompanyInfo from './components/CompanyInfo/CompanyInfo';
import Footer from './components/Footer/Footer';
import Parallax from './components/Parallax/Parallax';
import data from './data.json';

class App extends React.Component {
  render() {
    return (
      <IntlProvider locale="en" messages={data}>
        <div>
          <Home />
          <Navbar />
          <Parallax bg="./images/spider-office-1.jpg" />
          <About />
          <Parallax bg="./images/spider-office-2.jpg" />
          <Service />
          <Parallax bg="./images/spider-office-3.jpg" />
          <Technology />
          <Parallax bg="./images/spider-office-4.jpg" />
          <Client />
          <Parallax bg="./images/spider-office-5.jpg" />
          <Career />
          <Parallax bg="./images/spider-office-6.jpg" />
          <Contact />
          <Parallax bg="./images/spider-office-7.jpg" />
          <CompanyInfo />
          <Footer />
        </div>
      </IntlProvider>
    );
  }
}

export default App;
