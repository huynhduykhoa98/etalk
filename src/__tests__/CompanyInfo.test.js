/* eslint-disable linebreak-style */
import React from 'react';
import Enzyme, { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import CompanyInfo from '../components/CompanyInfo/CompanyInfo';
import App from '../App';

Enzyme.configure({ adapter: new Adapter() });
it('test render', () => {
  const wrapped = shallow(<App />);
  expect(wrapped.find(CompanyInfo).length).toEqual(1);
});
