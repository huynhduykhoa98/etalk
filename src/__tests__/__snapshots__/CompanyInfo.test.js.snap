// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`changes the class when hovered 1`] = `
<section>
  <div
    className="container"
  >
    <a>
      <h2>
        SPIDER COMPANY LIMITED
      </h2>
    </a>
    <div
      className="main"
    >
      <p>
        Address:
        <a
          href="https://goo.gl/maps/h1ygYS3HS8xfzrPS8"
          rel="noreferrer"
          title="Google maps"
        >
           8 Hoang Minh Giam Street, Ward 9, Phu Nhuan District, Ho Chi Minh City, Vietnam
        </a>
      </p>
      <p>
        Mobile:
        <a
          href="tel:+84905549909"
          title="Call us"
        >
          +84 90 5549909
        </a>
      </p>
      <p>
        Email:
        <a
          href="mailto:contact@spider.vn"
          title="Contact us"
        >
          contact@spider.vn
        </a>
      </p>
      <p>
        Website:
        <a
          href="//spider.vn"
          target="_blank"
          title="Visit our website"
        >
          spider.vn
        </a>
      </p>
      <p>
        Skype:
        <a
          href="skype:len.nguyenvan?chat"
          title="Chat with us"
        >
          len.nguyenvan
        </a>
      </p>
    </div>
  </div>
</section>
`;
